### ISSUE TITLE
Page contents are not reloaded when navigating with the browser's back/forward buttons

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-code-manager#932]
**STR:**

1, Open an extension's browse page  - see https://code.addons-dev.allizom.org/en-US/browse/502868/versions/1688187/
2. Open a few of the available files, one by one
3. Try to navigate to previously opened files with the browser's back button

**Actual result:**
The file path in the URL changes, but the user remains on the same page => page contents are not refreshed

**Expected result:**
The page contents should be refreshed when using the browser's keys to navigate to previously opened files

**Notes:**
- the Compare page will show a similar behavior, the only difference here is that the code overview section is updated, but, like in the browse page, the side panels are not refreshed
- reproduced on dev and stage with Fx67, Win10x64

![2019-07-09_14-17-22](https://user-images.githubusercontent.com/31961530/60883985-652e5a80-a254-11e9-870a-ddb7e5d6fa2a.gif)

**Comment by kumar303** [comment]
Maybe we don't have the `history` module [configured right](https://github.com/mozilla/addons-code-manager/blob/cb35fdfbca9619236e2f988178c17826750dce36/src/index.tsx#L15).

**Comment by mirefly** [comment]
I would like to work on this issue for the next. It looks like that collaborators are not allowed to assign an issue, so I will just leave a message before starting to work on a new issue.

**Comment by AlexandraMoga** [comment]
Verified fixed on -dev for the browse view:

![2020-01-03_13-58-55](https://user-images.githubusercontent.com/31961530/71722444-6c17dd80-2e31-11ea-9a71-4b5f348cb282.gif)
